auth-service/src/main/java/hcmus/alumni/authservice/model/PasswordHistoryModel.java

```diff
@@ -0,0 +1,61 @@
+package hcmus.alumni.authservice.model;
+
+import java.util.Date;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Entity
+@Table(name = "[password_history]")
+@AllArgsConstructor
+@Data
+public class PasswordHistoryModel {
+	@Id
+    @Column(name = "id", length = 36, nullable = false)
+    private String id;
+
+    @Column(name = "user_id", length = 36, nullable = false)
+    private String userId;
+
+    @Column(name = "password", length = 60, nullable = false)
+    private String password;
+
+    @Column(name = "is_auto_generated", nullable = false)
+    private boolean isAutoGenerated;
+
+    @CreationTimestamp
+    @Column(name = "created_at")
+    private Date createdAt;
+    
+    @Column(name = "updated_at")
```

Comment:
```suggestion
    @UpdateTimestamp
    @Column(name = "updated_at")
```
tự động cập nhật thời gian khi insert/ update data